code/components/voip-server-mumble/src/client.cpp

```diff
@@ -941,13 +941,15 @@ int Client_voiceMsg(client_t *client, uint8_t *data, int len)
 	} else if ((vt = Voicetarget_get_id(client, target)) != NULL) { /* Targeted whisper */
 		int i;
 		channel_t *ch;
+		std::map<int, bool> targeted_channels;
```

Comment:
This is a very hot code path, allocating dynamically multiple times in here could be problematic. Since this is a very small map most likely, using a stack-allocated eastl::fixed_map with, say, a 256-object max (and overflow enabled) might be more suited? You might have to add the vendor component, however.